alfresco-bom-repo/eu/xenit/alfresco/alfresco-community-bom/6.1.2-ga/alfresco-community-bom-6.1.2-ga.pom

```diff
@@ -256,8 +256,8 @@
       </dependency>
       <dependency>
         <groupId>org.apache.geronimo.specs</groupId>
-        <artifactId>geronimo-jta_1.0.1b_spec</artifactId>
-        <version>1.0.1</version>
+        <artifactId>geronimo-jta_1.1_spec</artifactId>
```

Comment:
did this while trying to fix an error produced by some other package"hibernate", it can be reverted but version 1.0.1b is not a release .your call?